app/src/main/java/com/nmakademija/nmaakademija/App.java

```diff
@@ -29,6 +29,7 @@ public void onCreate() {
         refWatcher = LeakCanary.install(this);
 
         setFirebaseOptions();
+        initFacebookSdk();
```

Comment:
I am pretty sure that with newest Facebook SDK there is no need to register inside Activity